<launch>
  <!--<arg name="uri" default="radio://0/50/1M" />-->
  <arg name="uri" default="radio://0/40/2M" />
  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="autorepeat_rate_" default="100" />

  <include file="$(find crazyflie_driver)/launch/crazyflie_server.launch">
  </include>


  <!--<param name="crazyflie/pid_rate/yaw_kp" value="200"/>-->
  <include file="$(find crazyflie_driver)/launch/crazyflie_add.launch">
  <arg name="uri" value="$(arg uri)" />
  <arg name="tf_prefix" value="cf1" />
  </include>

  <node name="joy" pkg="joy" type="joy_node" output="screen" >
  <param name="dev" value="$(arg joy_dev)" />
  <param name="autorepeat_rate" value="$(arg autorepeat_rate_)" />
  </node>

  <node name="esti_pub" pkg="estimate" type="estimate_pub" output="screen" >
  </node>

  <node name="generate_reference" pkg="generate_reference" type="generate_reference" output="screen" >
  </node>

  <node name="mpc_main" pkg="mpc" type="formation_control" output="screen" >
  </node>

  <include file="$(find crazyflie_demo)/launch/ps3_test.launch">
  </include>

</launch>
